### ISSUE TITLE
[feature] Read nvidia temperature from nvidia driver

### ISSUE THREAD

**Issue by TheTesla** [lm-sensors/lm-sensors#232]
Nvidia provides the GPU temperature via proprietary driver. It should be integrated in lm-sensors to provide it for fancontrol and pwmconfig.

**Comment by mira-roza** [comment]
yes please, I don't find something that does like fancontrol that has support nvidia temperature sensor

**Comment by MikeSesketcharba** [comment]
i still want this
